### ISSUE TITLE
[a11y]: Certificate Status Callout Box

### ISSUE THREAD

**Issue by peterkulko** [openedx/frontend-app-learning#1884]
## Description

Certificate Status should be marked up as a heading level 2. 

PR: https://github.com/openedx/frontend-app-learning/pull/1835
